san/sanbase_graphql.py

```diff
@@ -1,6 +1,6 @@
 import iso8601
 import datetime
-from san.pandas_utils import merge
```

Comment:
It had to be done, so that inspect.getmembers() doesn't count 'merge' as a function from this file.